content/source/docs/enterprise/private/automating-the-installer.html.md

```diff
@@ -146,7 +145,11 @@ The following apply to every installation:
 
 #### The following apply if you've chosen `external` for `production_type`:
 
-- `postgres_url` — (Required) the database URL
+- `pg_user` — (Required) PostgreSQL user name to connect as
+- `pg_password` — (Required) The password for the PostgreSQL user
+- `pg_netloc` — (Required) The `hostname:port` of the target PostgreSQL server
```

Comment:
This makes sense to me, but I wonder if it's better to be extremely explicit and say "The hostname and port of the target...in the format `hostname:port`."